features/api-layer/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -259,4 +270,13 @@
     </bean>
   </service>
 
+  <!-- Topology -->
```

Comment:
Since we are tied to `userDefinedLinkDao` being available for `UserDefinedLinkDaoImpl` to be started and in turn this bundle, If I  install `opennms-integration-api-features`  without enabling Enlinkd then this feature will fail which would be a problem for all other integration api that depend on this feature.

I think we have this problem with all other modules like  `Pollerd` , `Collectd` as well if they are not enabled.

Reply:
The `userDefinedLinkDao` and `userDefinedLinkTopologyService` services are actually referenced as `optional` to avoid this problem.